src/website/webview/templates/suggestion_list.html

```diff
@@ -24,15 +24,21 @@ <h1>Automatically generated suggestions</h1>
           {% severity_badge object.base_severity %}
         </div>
 
-        <h2>{{ object.package_name }}</h2>
-
         <details class="description">
           <summary>
-              {% if object.title %}{{ object.title }}{% else %}{{ object.description|truncatewords:10 }}{% endif %}
+            <h2>
```

Comment:
No need to make it a section or wrap it at all
```suggestion
```

Reply:
Currently we need this to distinguish from the "Show more" styling. And it *is* the title, so might have some benefit for accessibility tools?

Reply:
> Currently we need this to distinguish from the "Show more" styling

That's not true, the `:after` content can be styled completely separately.

> And it is the title, so might have some benefit for accessibility tools?

The schema field is called "title" but in reality it's a one-sentence summary. `<summary>` is the correct semantics here (takes of pointed hat and goes back to work)

Reply:
Done